### DISCUSSION TITLE
Using spacy on gpus for inference

### DISCUSSION THREAD

**Discussion by debraj135** [explosion/spaCy#7852] [Help: Coding & Implementations] [answered]
Is it possible to use spaCy with gpus for inference? If so, is there any documentation on how to go about doing this?

**Comment by svlandeg** [comment]
Hi! As a maintenance note: this is a good question for the discussions forum, so I'll move it there. The original issue will be closed and will link through to the open discussion thread.

**Comment by svlandeg** [comment, marked as answer]
Yes, you sure can! 
You can find the documentation for `prefer_gpu` and `require_gpu` here: https://spacy.io/api/top-level#spacy.prefer_gpu

**Comment by debraj135** [comment]
Thank you! Appreciate the prompt response
